apps/server/src/routes/agent/mcp.ts

```
@@ -0,0 +1,475 @@
+import { getZeroAgent } from '../../lib/server-utils';
+import { anthropic } from '@ai-sdk/anthropic';
+import { connection } from '../../db/schema';
+import { FOLDERS } from '../../lib/utils';
+import { env } from 'cloudflare:workers';
+import { eq, and } from 'drizzle-orm';
+import { McpAgent } from 'agents/mcp';
+import { createDb } from '../../db';
+import { generateText } from 'ai';
+import z from 'zod';
+
+export class ZeroMCP extends McpAgent<typeof env, {}, { userId: string }> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)